pages/s/[org]/[repo]/index.tsx

```diff
@@ -248,36 +248,22 @@ export default function RepoPage({ repoData, ogImageUrl }: RepoPageProps) {
                       <AddToWorkspaceDrawer type="sbom" repository={repoData.full_name} />
                     </>
                   ) : (
-                    <div className="flex flex-col items-end gap-2 w-fit">
-                      <Button
-                        variant="primary"
-                        onClick={() => {
-                          posthog.capture("Repo Pages: clicked 'Add to Workspace'", { repository: repoData.full_name });
-                          setIsAddToWorkspaceModalOpen(true);
-                        }}
-                        className="shrink-0 items-center gap-3 w-full"
-                      >
-                        <MdWorkspaces />
-                        Add to Workspace
-                      </Button>
-                      <Tooltip
-                        content={
-                          <div className="grid gap-2">
-                            <p>Create a workspace from the software bill of materials (SBOM) for this repository</p>
-
-                            <a
-                              href="https://opensauced.pizza/docs/features/repo-pages/#create-a-workspace-from-sbom"
-                              className="underline"
-                              target="_blank"
-                            >
-                              Learn more...<span className="sr-only"> about SBOM</span>
-                            </a>
-                          </div>
-                        }
-                      >
-                        <Button
-                          variant="dark"
-                          onClick={() => {
+                    <SplitButton
```

Comment:
The split button works, but it looks like now that it's being used, it causes some layout shift.

![CleanShot 2024-08-28 at 18 05 54](https://github.com/user-attachments/assets/8ab0e0db-bf4e-4b76-82ea-6b4e3b4e474c)